draft-patton-cfrg-vdaf.md

```diff
@@ -333,6 +377,8 @@ Syntactically, a VDAF is made up of the following algorithms:
   parameters are generated once and reused across multiple VDAF evaluations. The
   verification parameter MUST NOT be revealed to the clients.
 
+<!-- RLB: Who holds the private key?  One aggregator?  All aggregators? -->
```

Comment:
This is something that's a bit in-flux right now. I think where we're going to land is that the client gets a "public key" (the "public parameter") and each aggregator gets its own "secret key" (the "verification parameter").